README.md

```diff
@@ -0,0 +1,33 @@
+# Movie Recommendation system application using Taipy
+- In the application, the user's information is secured as `user_id`.
+- The app recommends **top_k** movies for a user via user_id, also give the **top_k actual** movies that the user is interested in.
+- There are 2 models used to make recommendations: *kNN* and *MF*
+- There are two metrics to evaluate the performance of the model: *recall* and *precision*. These are calculated automatically after making recommendations
+
+## Run the application on a local machine
+- Step 1: Clone this repo to the computer.
+At the conmmand prompt, type <br/>
+<code>git clone https://github.com/caotuannghia20/recsys-taipy-app.git </code>
+ - Step 2: Download the dataset [here](https://drive.google.com/file/d/1E_6jSGexFkjQmECkXEoegsp1KzoeOS4C/view?usp=sharing)
```

Comment:
The link should lead to GroupLens page. Then there should be a section to describe how to split the data to train and test the recommendation model.